editor/script/script_editor_plugin.cpp

```diff
@@ -3499,6 +3499,14 @@ void ScriptEditor::_make_script_list_context_menu() {
 	context_menu->popup();
 }
 
+void ScriptEditor::_script_name_label_gui_input(const Ref<InputEvent> &p_event) {
+	ERR_FAIL_COND(p_event.is_null());
+	Ref<InputEventMouseButton> mb = p_event;
+	if (mb.is_valid() && mb->is_pressed() && mb->get_button_index() == MouseButton::LEFT && mb->is_double_click()) {
+		script_list->ensure_current_is_visible();
```

Comment:
Note that the side panel containing the script list might be hidden, probably it should be auto shown here in such case?